rcljava/src/main/java/org/ros2/rcljava/publisher/statuses/OfferedDeadlineMissed.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.ros2.rcljava.publisher.statuses;
+
+import java.util.function.Supplier;
+
+import org.ros2.rcljava.common.JNIUtils;
+import org.ros2.rcljava.events.PublisherEventStatus;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class serves as a bridge between a rmw_offered_deadline_missed_status_t and RCLJava.
+ */
+public class OfferedDeadlineMissed implements PublisherEventStatus {
+  public int totalCount;
+  public int totalCountChange;
```

Comment:
I missed this for other statuses in previous PRs, but I think these member variables should be private and we can add getters for them. I think this will provide a bit better encapsulation (e.g. if symbol names change in the future, it's easier to deprecate a method than a member variable).

I don't think making them private has any impact on the native code setting their values.

Reply:
I think that passive data types only carrying data are a valid citizen in oop.
When the class doesn't have any invariant (relationship between fields) there's not much value on using private data, setters and getters.

> if symbol names change in the future, it's easier to deprecate a method than a member variable

If we change the name of one of the members we're going to break all other clients, so I don't think that's going to happen.

---

I realized (late), that the generated java bindings for ROS messages doesn't expose the data and provides getters/setters instead.
Though I don't like that, maybe being consistent in event statuses is better.

If would prefer doing this change after merging all remaining PRs in the feature branch.

Reply:
Merged, we can solve this issue before merging the feature branch on master.

Reply:
I'm ambivalent about it. Maybe we just leave it as-is and bring it up again when we open a PR upstream.

I'm not sure what the reasoning was to generating getters and setters for ROS messages; this too may be something we can change later upstream.

Reply:
Sounds good